frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -733,328 +714,288 @@ export function HandMarkedPaperBallot({
   );
 
   return (
-    <Ballot aria-hidden data-ballot ref={ballotRef}>
-      <div className="ballot-footer">
-        <PageFooter>
-          {isAbsentee && (
-            <AbsenteeFooter>
-              <Text bold as="span">
-                Absentee Ballot
+    <React.Fragment>
+      <div ref={pagedBallotRef} />
```

Comment:
That's just how PagedJS works. It renders the chunked content to a target element. 

Are you asking why we couldn't just render to the same `div` as the unpaged ballot?

Reply:
Ah ok thanks for that context.

Yes I was wondering why it couldn't just render to the same `div`.

Either way, some explanatory comments could be helpful in understanding the flow of action in this component.

Reply:
In this case, we can't target the same div because it is currently styled with `display: none;`. So unless we change the class names too, it seems easier to me to just target the a different div. I also like it for debugging, so in the inspector we can look at the paged and unpaged versions.

Reply:
Ok, sounds fine to me